client/my-sites/plans/current-plan/current-plan-thank-you/search-thank-you.tsx

```diff
@@ -1,32 +1,22 @@
 import { Button } from '@automattic/components';
 import { useTranslate } from 'i18n-calypso';
-import React, { ReactElement, useMemo } from 'react';
-import { useSelector } from 'react-redux';
-import versionCompare from 'calypso/lib/version-compare';
-import { getSelectedSite } from 'calypso/state/ui/selectors';
+import React, { ReactElement } from 'react';
 import ThankYou, { ThankYouCtaType } from './thank-you';
 
 const ThankYouCta: ThankYouCtaType = ( {
 	dismissUrl,
-	jetpackVersion,
+	jetpackSearchCustomizeUrl,
 	recordThankYouClick,
-	siteAdminUrl,
 } ) => {
 	const translate = useTranslate();
 	return (
 		<>
 			<Button
 				primary
-				href={
-					jetpackVersion && versionCompare( jetpackVersion, '8.4', '<' )
```

Comment:
Is this pre 8.4 logic no longer required?

Reply:
According to site profiles, only 0.15% of our customers would ever encounter this messaging. 

Also, the utility of this notice is questionable given that:
1) There's no way to purchase Search via wp-admin on Jetpack versions < 8.4.
2) The only action we can recommend is to upgrade their Jetpack plugin.

Given these tradeoffs, I opted for simplifying our business logic here.